tensorflow/python/kernel_tests/map_ops_test.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Tests for TensorMap ops."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from absl.testing import parameterized
+from tensorflow.python.eager import backprop
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import errors
+from tensorflow.python.framework import test_util
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import map_ops
+from tensorflow.python.platform import test
+
+from tensorflow.python.util.lazy_loader import LazyLoader
+control_flow_ops = LazyLoader("control_flow_ops", globals(),
```

Comment:
Why do you need to use LazyLoader here? I would not expect the test to be part of a circular dep so the lazy loading should be avoidable.

Reply:
Ah, TensorList uses LazyLoader, but you're right it works here without

Reply:
Yes, no test should use LazyLoader - that is a tool only useful for circular dependencies, and nothing else should depend on tests.